types/xregexp/index.d.ts

```diff
@@ -83,7 +84,7 @@ declare namespace OuterXRegExp {
     function uninstall(options: Object): void;
     function uninstall(options: string): void;
 
-    function union(patterns: string[], flags?: string): RegExp;
+    function union(patterns: string[] | RegExp[], flags?: string): RegExp;
```

Comment:
@RyanCavanaugh Sorry, I don't understand _it need not be uniform_ phrase. Could you tell it in other words?

Should I only change `string[] | RegExp[]` entry to the `(string | RegExp)[]`?